### DISCUSSION TITLE
Allow setting `brand.logo` to `false` to hide the logo?

### DISCUSSION THREAD

**Discussion by matthewjumpsoffbuildings** [laravel/nova-issues#4867] [Ideas & Feature Requests]
I don't want a logo on the top left next to the app name. With Nova 3 I just customised some blade templates, but now for the life of me I cannot turn off the damn logo.

It should be that setting the `brand.logo` property to `false` in the nova config file just hides the entire logo

**Comment by wamesro** [comment]
Add a transparent 1px SVG

but of course FALSE would be better
